com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -616,6 +616,9 @@ public void mount(IResourceScanner resourceScanner) throws IOException, CompileE
         this.fileSystem.clearMountPoints();
         this.fileSystem.addMountPoint(new ClassLoaderMountPoint(this.fileSystem, "builtins/**", resourceScanner));
         Map<String, File> libFiles = LibraryUtil.collectLibraryFiles(getLibPath(), this.libUrls);
+        if (libFiles == null) {
+            throw new CompileExceptionError("Missing libraries folder. You need to run the 'resolve' command first!");
+        }
```

Comment:
Another issue: If you try to build before resolving the libraries, you got an ugly nulll pointer callstack, with no info about the problem.